csm_web/frontend/src/components/App.js

```diff
@@ -40,13 +40,32 @@ export default class App extends React.Component {
   }
 }
 
+const applyDarkMode = isDarkMode => {
+  window.localStorage.setItem("darkMode", isDarkMode);
+  document.getElementById("app").className = isDarkMode ? "dark-mode" : "";
+  document.body.style.backgroundColor = isDarkMode ? "#1c1d1e" : "white";
+};
+
```

Comment:
this is a pretty hacky way to do this. a very non-react way to add dark mode. look up dark mode with react and change your implementation to follow one of those tutorials

Reply:
Resolved in latest commit by pulling theme classes into the React tree